wordstats/language_data/t2l/code/README.md

```diff
@@ -0,0 +1,62 @@
+### cleaning_words.py
+
+If it is required to add words of a new language it is needed to follow the next steps:
+
+1. Download the aspell dictionary of that language (http://ftp.gnu.org/gnu/aspell/dict/) and follow the instructions. Basically you have to download the .tar.bz2 file (the 5 is the newest) and then:
+
+	-Unzip the folder
+
+	-Go with the terminal to that folder.
+
+	-Execute ./configure
+
+	-Execute make
+
+	-Execute make install
+
+	-Execute preunzip 'lang'.cwl (e.g: es.cwl)
+
+You will obtain a .wl file with the dictionnary.
+
+However there will be a script which will do this automatically, but if the result is not what you expect, then you can do it manually.
+
+2. It is needed to create a new .txt file and copy the dictionary. Python does not recognize .uw files. After that move the dictionary to the Dictionary folder.
+
+3. The input of the script will be the code of the language you want to obtain the words from.
+
+NOTE: If there is no freq list inside /hermitdave/2016/ then it will be needed to add a new txt file with the 50k most frequent words of a language. In order to avoid code modifications, it will be necessary to add that file in the next way. First add a new folder in /2016/ with the code of the language (e.g. for Spanish 'es') and inside that folder the txt file in the next form: 'code_lang'_50k.txt (e.g for spanish 'es_50k.txt').
+
+NOTE 2: If there are changes in the structure of the names or the structure of directories, the code may need changes.
+
+
+### translation.py and query.py
+
+To compute the translations Glosbe is used. After do some request (around 800 in general) the IP will be blocked and the program will be sleeping until it is unblocked. After that it will continue.
+
+To change the orig language and the translation language it is needed to change the variables FROM_CODE and DEST_CODE to the wanted languages codes.
+
+IMPORTANT: Once the IP is blocked it is needed to go to the glosbe web-site, do a query and check that you are human. Once this is done, the IP will be unblocked and the program will continue running
+
+NOTE: If there are changes in the structure of the names or the structure of directories, the code may need changes.
+
+
+### getDictonary.sh, languagelist.awk and tarselect.awk
+
+These files form the script to download automatically the dictionary the user wants and it is put in the dictionaries folder directly. It is only needed to run the .sh file and then, there will appear the neccessary instructions.
```

Comment:
would be cool to have these three things, who belong together in their own folder e.g. `./dict_retrieval` or even `get_dictionary`.